Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.CssEditor/scripts/CssEditor.js

```diff
@@ -97,26 +98,82 @@ define([
             if (params.settings.isHost) {
                 getPortals();
             }
-            getStyleSheet();
+            
+            initCssEditor();
 
-            cssEditor = codeEditor.init($panel.find('textarea'), { mode: 'css' });
             var panelHeight = $('#CssEditor-panel').height();
             if (panelHeight > 400) {
-                cssEditor.setSize("100%", panelHeight - 330);
+                $('#monaco-editor').height(panelHeight - 400);
             }
 
             if (typeof callback === 'function') {
                 callback();
             }
 
-            cssEditor.on("blur", function(cm) {
+            $('#monaco-editor').on("blur", function(cm) {
                 cm.save();
                 return true;
             });
 
             viewModel.portal.subscribe(portalChanged);
         };
 
+        var initCssEditor = function() {
+            var monacoEditorLoaderScript = document.createElement('script');
+            monacoEditorLoaderScript.type = 'text/javascript';
+            monacoEditorLoaderScript.src = '/Resources/Shared/components/MonacoEditor/loader.js';
```

Comment:
This PR is maintaining the MonocoEditor minified source code under our shared components. This will make it difficult to maintain the version of the editor DNN is using. Did you try using any of the npm packages that exist for MonacoEditor?

Reply:
The CSS Editor extension for the Persona Bar does not use React and doesn't have a Webpack config at this point, so this approach was used as a first implementation.  The plan is to switch it to use a version from npm.

@david-poindexter @valadas @mitchelsellers Do we want to avoid adding these scripts to the Shared Resources folder if we're planning to remove them soon?  We don't want to give developers the impression that they're intended for broader use, right?

Reply:
@bdukes Maybe for this version it would make sense to include under the /Dnn.CssEditor extension until we include this for usage in other locations and properly pull it in?  @david-poindexter thoughts?

Reply:
Thanks for the clarification on the CSS Editor, I thought something was odd when I didn't see any React code. 

Is there an item already on the backlog for updating this to use an NPM package instead of adding the minified source code? Was there a barrier to getting it working?

Reply:
@ahoefling please refer to #4132 - this is a "stepping stone" solution.  This is obviously not the "right way" to do this, but the current CssEditor module in the PersonaBar is not a proper solution with package management.  Therefore, that is a task for a future PR.